<ui version="4.0" >
 <class>MWDiag</class>
 <widget class="QWidget" name="MWDiag" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lbIFile" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>85</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>Hard Mask File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4" >
    <widget class="QLineEdit" name="leIFile" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6" >
    <widget class="QPushButton" name="pbIFile" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lbOFile" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>85</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>Output Mask File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4" >
    <widget class="QLineEdit" name="leOFile" />
   </item>
   <item row="1" column="5" >
    <spacer name="horizontalSpacer_14" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6" >
    <widget class="QPushButton" name="pbOFile" >
     <property name="text" >
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lbSignificance" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>85</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>Errorbar Criterion</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="leSignificance" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>63</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>63</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <spacer name="horizontalSpacer_15" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4" colspan="2" >
    <widget class="QCheckBox" name="ckAngles" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>No Solid Angles</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7" >
    <widget class="QGroupBox" name="gbIndividual" >
     <property name="title" >
      <string>Individual White Beam Tests</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>2</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>2</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lbWBV1" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>White Beam Van1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lbHighAbs" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>High Counts</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leHighAbs" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="validatorSpace_6" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>20</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="lbLowAbs" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Low Counts</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8" >
       <widget class="QLabel" name="validatorSpace_4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>20</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lbHighMed" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Median Test High</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leHighMed" />
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="validatorSpace_5" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>20</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="3" >
       <widget class="QLineEdit" name="leLowMed" />
      </item>
      <item row="2" column="8" >
       <widget class="QLabel" name="validatorSpace_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>20</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4" >
       <widget class="QLineEdit" name="leWBV1" />
      </item>
      <item row="1" column="4" colspan="3" >
       <widget class="QLineEdit" name="leLowAbs" />
      </item>
      <item row="0" column="5" >
       <widget class="QLabel" name="validatorSpace_9" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>20</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6" >
       <widget class="QPushButton" name="pbWBV1" >
        <property name="text" >
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="lbLowMed" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Median Test Low</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="7" >
    <widget class="QGroupBox" name="gbVariation" >
     <property name="title" >
      <string>Efficiency Variation Test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="margin" >
       <number>2</number>
      </property>
      <item row="1" column="2" >
       <spacer name="horizontalSpacer_7" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>23</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lbVariation" >
        <property name="text" >
         <string>Proportional change criterion</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leVariation" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="validator_label" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="7" >
    <widget class="QGroupBox" name="gbBackTest" >
     <property name="title" >
      <string>Background check</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="ckDoBack" >
        <property name="text" >
         <string>Check backgrounds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3" >
       <widget class="QLabel" name="lbAcceptance" >
        <property name="text" >
         <string>Acceptance Factor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QLineEdit" name="leAcceptance" />
      </item>
      <item row="0" column="6" >
       <widget class="QLabel" name="validator_label_3" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lbStartTime" >
        <property name="text" >
         <string>Start TOF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="leStartTime" />
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="lbEndTime" >
        <property name="text" >
         <string>End TOF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QLineEdit" name="leEndTime" />
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QCheckBox" name="ckZeroCounts" >
        <property name="text" >
         <string>Reject zero background</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="6" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="valErr" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QLabel" name="valInmsk" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>*</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
